src/main/java/com/hotels/bdp/circustrain/bigquery/partition/BigQueryPartitionGenerator.java

```diff
@@ -99,11 +102,16 @@ private Table createPartitionInBigQuery(
     return part;
   }
 
-  private ExtractionUri scheduleForExtraction(Table table, Partition partition) {
+  private ExtractionUri scheduleForExtraction(
```

Comment:
To clarify the types in the method arguments perhaps we could rename them like so:

table -> bigQueryTable
partition -> hivePartition
sourceTableAsHive -> hiveTable

(and then put the Hive table before the partition)